setup.py

```diff
@@ -76,7 +76,6 @@ def find_version(*file_paths):
         "emcee<3",
         "powerbox>=0.5.7",
         "cached_property",
-        "21cmFAST>=3.0.0dev",
```

Comment:
I think this should be changed to `21cmFAST @ @ git+git://github.com/21cmFAST/21cmFAST`

Reply:
Ah yes, that's true... I've tried multiple formats, the link should just have this `egg` part also:
```suggestion
        "21cmFAST @ git+git://github.com/21cmfast/21cmFAST#egg=21cmFAST",
```

Reply:
Ah OK, that looks right!